File: src/main/java/org/c3lang/intellij/psi/C3FileStub.java
package org.c3lang.intellij.psi;

import com.intellij.psi.stubs.PsiFileStubImpl;

public class C3FileStub extends PsiFileStubImpl<C3File> {

    public C3FileStub(C3File file) {
        super(file);
    }
}

File: src/main/java/org/c3lang/intellij/stubs/C3FileStub.kt
package org.c3lang.intellij.stubs

import com.intellij.psi.stubs.PsiFileStubImpl
import org.c3lang.intellij.psi.C3File

class C3FileStub : PsiFileStubImpl<C3File> {

    constructor(file: C3File) : super(file)

}